.github/workflows/commit_epm.yml

```diff
@@ -69,5 +69,24 @@ jobs:
         sed -i '145 c \"enclave.runtime.path\": \"/var/run/rune/liberpal-skeleton-v3.so\",' config.json;
         rune --debug run ra"
 
+    - name: Run skeleton v3 with no epm
+      if: always()
+      run: docker exec $rune_test bash -c "docker run -i --rm --runtime=rune -e ENCLAVE_TYPE=intelSgx -e ENCLAVE_RUNTIME_PATH=/usr/lib/liberpal-skeleton-v3.so -e ENCLAVE_RUNTIME_ARGS=debug,no-epm -e ENCLAVE_RUNTIME_LOGLEVEL="info" skeleton-enclave"
+
+    - name: Run skeleton v3 bundle with no epm
+      run: |
+        docker exec $rune_test bash -c "mkdir rootfs;
+        docker create --name skeleton-enclave skeleton-enclave;
+        docker export skeleton-enclave | tar -C rootfs -xvf -;
+        cp /etc/resolv.conf rootfs/etc/resolv.conf;
+        mkdir -p /var/run/rune;
+        cp /usr/lib/liberpal-skeleton-v3.so /var/run/rune/liberpal-skeleton-v3.so;
+        rune spec;
+        sed -i '4 c \"terminal\": false,' config.json;
+        sed -i '16 c \"cwd\": \"\/\",' config.json;
+        sed -i '143 c \"enclave.runtime.args\": \"debug, no-epm\",' config.json;
+        sed -i '145 c \"enclave.runtime.path\": \"/var/run/rune/liberpal-skeleton-v3.so\",' config.json;
+        rune --debug run ra"
```

Comment:
There is no need to create a new bundle. Just use the previous bundle and change `enclave.runtime.args` from `debug `  to `debug , no epm`

Reply:
thanks. Done